SwiftWisdom/Core/UIKit/NibLoadableView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NibLoadableView.swift
+//  SwiftWisdom
+//
+//  Created by Christian Howland on 3/31/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+/**
+ Subclassing NibLoadableView allows you to automate the loading of a Nib into a placeholder view inside
+ another Nib, using only the Custom Class attribute.
+ 
+ It removes the need to perform the following manual steps every time you want a Nib that can load into a placeholder View:
+
+ - Setting an outlet for the top level view
+ - Setting 'translatesAutoresizingMaskIntoConstraints' to false
+ - Adding constraints to fit the new view into its container
+ - Writing boilerplate Nib loading code.
+
+ The minimum amount of code required to use this would just be the inheritance line, i.e.:
+
+ " YourCustomView: NibLoadableView {} "
```

Comment:
@brightredchilli, What you said - that there would be 2 nested views of the same type - is correct. The use case I am trying to solve is the following: 
you have a custom view that has both a class definition AND its own nib, with outlets defined between the 2. In my experience, in order to get the custom view loading into a placeholder view within another nib, it is necessary to create the duplicate layer because you need to set the view outlet manually, load up the nib, etc. This solution aims to simplify that process.

Having said that, if you think there would be a way to approach the above situation without having the extra layer, I'd definitely be interested in your thoughts.